packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -2306,6 +2306,14 @@ const Conversation = WebexPlugin.extend({
       }));
     }
 
+    if (!params.isDefaultClassification && params.classificationId) {
```

Comment:
Please add these additional params to the jsdoc on the params object on the `create` function near line 167.